Generate GitHub action broken due to permission changes

@rviscomi we now get this when generate GitHub action runs ([see here](https://github.com/HTTPArchive/almanac.httparchive.org/runs/638419637?check_suite_focus=true)):

```
Commiting changes
[master 177796e] Regenerated chapters
 3 files changed, 499 insertions(+), 1 deletion(-)
 create mode 100644 src/templates/ja/2019/chapters/media.html
remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: You're not authorized to push to this branch. Visit https://help.github.com/articles/about-protected-branches/ for more information.        
To https://github.com/HTTPArchive/almanac.httparchive.org
 ! [remote rejected] master -> master (protected branch hook declined)
error: failed to push some refs to 'https://github.com/HTTPArchive/almanac.httparchive.org'
##[error]Process completed with exit code 1.
```

This is even though this runs under my id.

This is because even those with write permissions to master (i.e. you and I) need to approve changes and that doesn't work with GitHub actions.

Can you and I be given permission to commit directly to master?

--- a/.github/workflows/generate_chapters.yml
+++ b/.github/workflows/generate_chapters.yml
@@ -34,8 +34,6 @@ jobs:
         echo "Checking git status"
         git status
         echo "Adding templates and content directory"
-        git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
-        git config --global user.name "Automatic Update"
         git add src/templates/*/*/chapters/
         git add src/templates/sitemap.xml
         echo "Commiting changes"

